app/src/main/java/com/cmput301f20t21/bookfriends/ui/profile/ProfileFragment.java

```diff
@@ -47,6 +56,30 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         emailAddress = view.findViewById(R.id.email);
         phoneNumber = view.findViewById(R.id.phone);
         editProfile = view.findViewById(R.id.image_edit);
+        name = view.findViewById(R.id.username);
+
+        // get the login information from firebase
+        User firebaseUser = AuthService.getInstance().getCurrentUser();
+        if (firebaseUser != null) {
+            String userId = firebaseUser.getUid();
+
+            //document reference
+            final DocumentReference docReference = FirebaseFirestore.getInstance().collection("users").document(userId);
```

Comment:
the User object firebaseUser already contains the information (username, email) that we need, you can get it by calling getUsername() and getEmail(). 

There's no need to get it from firestore again